src/Components/apis/useGetUser.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { useState, useEffect } from 'react';
+import axios from "axios";
+
+export const useGetUser = () => {
+    const [res, setRes] = useState({
+        success: false,
+        loading: true,
+        status: null,
+        data: null,
+        error: null
+    })
+
+    const getUser = async () => {
+        let token = localStorage.getItem("token")
+        console.log(token)
+        try {
+            const res = await axios.get("/api/profile/me", {
+                headers: {
+                    'x-auth-token': token,
+                    'Content-Type': 'application/json',
+                    'Access-Control-Allow-Origin': '*',
+                }
+            });
+            console.log(res)
```

Comment:
don't clutter your code with console statements remove them one you are done with debugging.